### PR TITLE
remove ckanext-googleanalyticsbasic, add ckanext-googleanalytics

### PR DESCRIPTION
**Pull request by FuhuXia**
`ckanext-googleanalyticsbasic` is not working in current release, as noted [here](https://github.com/GSA/data.gov/issues/3571).
this PR removes `ckanext-googleanalyticsbasic` and added `ckanext-googleanalytics`

note: `googleanalytics` has to be listed before `envvars` in the plugins list.

TODO: update googleanalytics.id  with real ones.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -47,7 +47,7 @@ CKAN_SMTP_MAIL_FROM=ckan@localhost
 # Extensions
 # TODO: Reenable pending https://github.com/GSA/data.gov/issues/3986
 # qa archiver
-CKAN__PLUGINS=envvars image_view text_view recline_view metrics_dashboard report ckan_harvester datajson_validator datajson_harvest geodatagov datagovtheme datagov_harvest geodatagov_miscs z3950_harvester arcgis_harvester geodatagov_geoportal_harvester waf_harvester_collection geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_metadata spatial_query spatial_harvest_metadata_api googleanalyticsbasic dcat dcat_json_interface structured_data datagovcatalog
+CKAN__PLUGINS=image_view text_view recline_view metrics_dashboard report ckan_harvester datajson_validator datajson_harvest geodatagov datagovtheme datagov_harvest geodatagov_miscs z3950_harvester arcgis_harvester geodatagov_geoportal_harvester waf_harvester_collection geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_metadata spatial_query spatial_harvest_metadata_api googleanalytics dcat dcat_json_interface structured_data datagovcatalog envvars
```
Comment (nickumia-reisys) [inline .env:50]: Note: this won't effect production.  Does the production config work?

File: ckan/requirements.in [hunk 2]
```diff
@@ -26,6 +26,9 @@ pika>=1.1.0,<1.3.0
 # requests>=2.11.1 # included in ckan core
 # six>=1.12.0 # included in dcat
 
+# ckanext-googleanalytics
+oauth2client
```
Comment (nickumia-reisys) [inline ckan/requirements.in:30]:
```suggestion
# ckanext-googleanalytics
# https://github.com/ckan/ckanext-googleanalytics/blob/master/requirements.txt
oauth2client
```

File: .env (hunk 1 above)
Comment (FuhuXia) [reply .env:50]: cf apps have `evnvars` listed at the end in the ckan.ini

**Comment by FuhuXia** [comment]
@dlennox24
We do not want to send data to google analytics from local/devvelop/staging. We can keep those fake IDs in those environments unless we add more real IDs later on.  So only the last line in `vars.prod.yml` needs to change.

**Comment by dlennox24** [comment]
@FuhuXia Good point! Should we test the tacking code in dev/stg to start and then remove them once we get confirmation that GA is receiving data?

**Comment by FuhuXia** [comment]
> @FuhuXia Good point! Should we test the tacking code in dev/stg to start and then remove them once we get confirmation that GA is receiving data?

Sure. let me add it into develop branch to have it deployed in development space. Will let you know once deployment is done so that you can verifiy.

**Comment by FuhuXia** [comment]
develop is **temporarily** using prod analytics id for testing. @dlennox24 

image

**Comment by dlennox24** [comment]
Data is flowing!

image

**Comment by jbrown-xentity** [comment]
@FuhuXia can you take a look at what's holding up this test? It's timed out, so the server isn't starting up for some reason... Might have to debug locally?

**Review by robert-bryson — APPROVED** [review]

**Pull request merged**